apps/playground-sveltekit/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "useTabs": false,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 100
+}
```

Comment:
Our default prettier config usually looks like this:

```js
  "prettier": {
    "semi": false,
    "singleQuote": false
  },
```

Let's keep the same for consistency's sake. :pray:

I would also prefer to just put it into the `package.json` file for fewer files in the root directory.